apps/api.namerank.io/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM --platform=linux/arm64 public.ecr.aws/lambda/python:3.11
```

Comment:
It would make sense to move the responsibility of setting `arm64` to the deployment script.